blocks/commerce-return-header/commerce-return-header.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-extraneous-dependencies */
+import { events } from '@dropins/tools/event-bus.js';
+import { Header, provider as uiProvider } from '@dropins/tools/components.js';
+import { CUSTOMER_RETURN_DETAILS_PATH, CUSTOMER_RETURNS_PATH } from '../../scripts/constants.js';
+
+// TODO - Will be refactored after https://jira.corp.adobe.com/browse/USF-1393 | https://atwix.atlassian.net/browse/EDS-426
+// Initialize
+import '../../scripts/initializers/order.js';
+
+export default async function decorate(block) {
+  block.innerHTML = '';
+
+  const headerContainer = document.createElement('div');
+  await uiProvider.render(Header, { title: 'Return' })(headerContainer);
+
+  if (window.location.href.includes(CUSTOMER_RETURN_DETAILS_PATH)) {
+    const link = document.createElement('a');
+
+    // TODO: Add i18n when appropriate functionality will be introduced in boilerplate
+    link.innerText = '< Back to all returns';
```

Comment:
We already have [placeholders](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/blocks/product-details/product-details.js#L141) in use by PDP. Probably best to put all copy we add into this placeholders file for now.

Reply:
Fixed ✅